packages/flutter_splash_tizen/LICENSE

```diff
@@ -0,0 +1,21 @@
+Copyright (c) 2021, Samsung Electronics Co., Ltd. All rights reserved.
```

Comment:
Please copy this file as it is:

https://github.com/flutter-tizen/plugins/blob/master/packages/tizen_app_control/LICENSE